src/Data.ts

```diff
@@ -1,16 +1,26 @@
-// the data for South Sudan (SSD), Palestinian Territories (PSE) and Kosovo (XK)
-// are dodgy so I've omitted them
-export const countries = ["AFG", "ALB", "AGO", "ARM", "AZE", "BGD", "BLZ",
-    "BEN", "BTN", "BOL", "BIH", "BFA", "BDI", "KHM", "CMR", "CPV", "CAF", "TCD",
-    "CHN", "COM", "CIV", "CUB", "DJI", "PRK", "COD", "EGY", "SLV", "ERI", "ETH",
-    "FJI", "GMB", "GEO", "GHA", "GTM", "GIN", "GNB", "GUY", "HTI", "HND", "IND",
-    "IDN", "IRQ", "KEN", "KIR", /*"XK",*/ "KGZ", "LSO", "LBR", "MDG", "MWI",
-    "MLI", "MHL", "MRT", "FSM", "MDA", "MNG", "MAR", "MOZ", "MMR", "NPL", "NIC",
-    "NER", "NGA", "PAK", /*"PSE",*/ "PNG", "PRY", "LAO", "PHL", "COG", "RWA",
-    "WSM", "STP", "SEN", "SLE", "SLB", "SOM", /*"SSD",*/ "LKA", "SDN", "SWZ",
-    "SYR", "TJK", "TZA", "TLS", "TGO", "TON", "TKM", "TUV", "UGA", "UKR", "UZB",
-    "VUT", "VNM", "YEM", "ZMB", "ZWE"];
-
+import {loadObjectFromJSONFile} from "./Utils";
+
+export const countries: string[] =
+    loadObjectFromJSONFile("./countryCodes.json");
+export const touchstones: string[] =
+    loadObjectFromJSONFile("./touchstones.json");
+export const supportTypes: string[] =
+    loadObjectFromJSONFile("./support.json");
+export const activityTypes: string[] =
+    loadObjectFromJSONFile("./activities.json");
+
+export const disVac = loadObjectFromJSONFile("./diseaseVaccines.json");
+export const diseases: string[] = [];
+export const vaccines: string[] = [];
+for (const k in disVac) {
+    diseases.push(k);
+    vaccines.concat(disVac[k]);
+}
+
+export const dates: {[code: string]: number[]} =
+    loadObjectFromJSONFile("./dates.json");
+
+// These can't be infered from the data so we have to hard code them somewhere
```

Comment:
Unfortunately now that countries are not hard-coded, it kind of breaks the approach below of working out which countries are Dove94, Gavi72, etc by filtering out the ones you know are not in those groups. If a new country gets added to the database, all the groups become inaccurate. 

Looks like we do now have this metadata about countries in the db though: https://vimc.github.io/montagu-db-docs/latest/tables/country_metadata.html

Reply:
I've had a look at the dataset I'm using from the science team and the necessary columns are there so I we should be able to read them from there.
[there's a lot of NAs in the columns so I'll have a talk with Xiang/Chris and ask if there's something subtle going on.]